document/core/exec/numerics.rst

```diff
@@ -1993,81 +1988,170 @@ each environment globally chooses a fixed projection for each operator.
 :math:`j` is a 8-bit int.
 
 .. math::
-   \begin{array}{@{}lcll}
-   relaxed\_swizzle\_lane(i^n, j) &=& [ i[j], i[j] ] & (\iff j < 16) \\
-   relaxed\_swizzle\_lane(i^n, j) &=& [ 0, 0 ] & (\iff \signed_8(j) < 0) \\
-   relaxed\_swizzle\_lane(i^n, j) &=& [ 0, i[j \mod n] ] & (\otherwise) \\
+   \begin{array}{@{}llcll}
+   & relaxed\_swizzle\_lane(i^n, j) &=& i[j] & (\iff j < 16) \\
+   & relaxed\_swizzle\_lane(i^n, j) &=& 0 & (\iff \signed_8(j) < 0) \\
+   \EXPROFDET & relaxed\_swizzle\_lane(i^n, j) &=& [ 0, i[j \mod n] ] & (\otherwise) \\
+   & relaxed\_swizzle\_lane(i^n, j) &=& 0 & (\otherwise) \\
    \\
-   relaxed\_swizzle(a^n, s^n) &=& rsl_0 \dots rsl_{n-1} \\
-   \qquad \where rsl_i &=& relaxed\_swizzle\_lane(a^n, s^n[i])
+   & \rswizzle(a^n, s^n) &=& rsl_0 \dots rsl_{n-1} \\
+   & \qquad \where rsl_i &=& relaxed\_swizzle\_lane(a^n, s^n[i])
    \end{array}
 
+
+.. _op-rtrunc:
+.. _op-rtrunc_u:
+
+:math:`\rtrunc^u_{M,N}(z)`
+..........................
+
 Relaxed truncate converts float to int, NaN and out of range values are
 hardware dependent.
 
 .. math::
-   \begin{array}{@{}lcll}
-   relaxed\_trunc^s_{M,N}(\pm \NAN(n)) &=& [ 0, -2^{N-1} ] \\
-   relaxed\_trunc^s_{M,N}(\pm q) &=& \truncs_{M,N}(\pm q) & (\iff -2^{N-1} - 1 < \trunc(\pm q) < 2^{N-1}) \\
-   relaxed\_trunc^s_{M,N}(\pm p) &=& [ \truncsats_{M,N}(\pm p), -2^{N-1} ] & (\otherwise) \\
-   \\
-   relaxed\_trunc^u_{M,N}(\pm \NAN(n)) &=& [ 0, 2^{N}-1 ] \\
-   relaxed\_trunc^u_{M,N}(\pm q) &=& \truncu_{M,N}(\pm q) & (\iff -1 < \trunc(\pm q) < 2^N) \\
-   relaxed\_trunc^u_{M,N}(\pm p) &=& [ \truncsatu_{M,N}(\pm p), 2^{N}-1 ] & (\otherwise) \\
+   \begin{array}{@{}llcll}
+   \EXPROFDET & \rtrunc^u_{M,N}(\pm \NAN(n)) &=& [ 0, 2^{N}-1 ] \\
+   \EXPROFDET & \rtrunc^u_{M,N}(\pm q) &=& \truncu_{M,N}(\pm q) & (\iff -1 < \trunc(\pm q) < 2^N) \\
+   \EXPROFDET & \rtrunc^u_{M,N}(\pm p) &=& [ \truncsatu_{M,N}(\pm p), 2^{N}-1 ] & (\otherwise) \\
+   & \rtrunc^u_{M,N}(\pm p) &=& \truncsatu_{M,N}(\pm p) & (\otherwise) \\
    \end{array}
 
+
+.. _op-rtrunc_s:
+
+:math:`\rtrunc^s_{M,N}(z)`
+..........................
+
+Relaxed truncate converts float to int, NaN and out of range values are
+hardware dependent.
+
+.. math::
+   \begin{array}{@{}llcll}
+   \EXPROFDET & \rtrunc^s_{M,N}(\pm \NAN(n)) &=& [ 0, -2^{N-1} ] \\
+   \EXPROFDET & \rtrunc^s_{M,N}(\pm q) &=& \truncs_{M,N}(\pm q) & (\iff -2^{N-1} - 1 < \trunc(\pm q) < 2^{N-1}) \\
+   \EXPROFDET & \rtrunc^s_{M,N}(\pm p) &=& [ \truncsats_{M,N}(\pm p), -2^{N-1} ] & (\otherwise) \\
+   & \rtrunc^s_{M,N}(\pm p) &=& \truncsats_{M,N}(\pm p) & (\otherwise) \\
+   \end{array}
+
+
+.. _op-rlaneselect:
+
+:math:`\rlaneselect_B(a^n, b^n, c^n)`
+.....................................
+
 Relaxed lane select is deterministic where all bits are set or unset in the
 mask. Otherwise depending on hardware, either only the top bit is examined, or
 all bits are examined (becomes a bitselect).
```

Comment:
```suggestion
Relaxed lane selection is deterministic when all bits are set or unset in the
mask. Otherwise depending on the host, either only the top bit is examined, or
all bits are examined (i.e., it becomes a bit select).
```